CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
-## 0.16.3-dev
+## 0.17.0-nullsafety-dev
```

Comment:
Not sure. I hesitate to show confidence that this code is "stable" as a `1.0.0` release because there are so many untested code paths. On the other hand I don't know that we have any plans to spend more time in here so it might not be worth waiting for work that won't happen and just bump the version now.

Reply:
Ya my concern would be I don't know the long term maintenance plan for this package, it might be more genuine to leave it at <1.0.0

Reply:
I'll keep it here for now. If we want to bump to `1.0.0` for the real null safety release I think we still have that option without much additional churn.